kronolith/js/kronolith.js

```diff
@@ -5997,6 +5999,18 @@ KronolithCore = {
         $('kronolithEventLinkExport').up('li').show();
         $('kronolithEventExport').href = Kronolith.conf.URI_EVENT_EXPORT.interpolate({ id: ev.id, calendar: ev.c, type: ev.ty });
 
+        /* Attendance and Status */
+        if (!ev.oy) {
+            $('kronolithEventStatus').disabled = true;
```

Comment:
Now that I see this in action, it doesn't make any sense to disable the status field. Either I allow attendees to edit their copy of the event or not.